css/main.scss

```diff
@@ -36,6 +36,7 @@ $flagsImagePath: "../images/";
 @import 'modals/invite/info';
 @import 'modals/screen-share/share-audio';
 @import 'modals/screen-share/share-screen-warning';
+@import 'modals/whiteboard';
```

Comment:
Oh, can't we really not add more stuff here?

Reply:
Oh sorry, should I move the style into the _base file or simply have a style object passed directly?